### DISCUSSION TITLE
Hola, Buen dia

### DISCUSSION THREAD

**Discussion by Henryterraza** [Launch-X-Latam/MisionFrontEnd#443] [Team Posts]
Con una duda, nos deben a parecer dos grupos que es fron-end y back-end, segun las instrucciones del respositorio de github del OnBoarding deben ser dos y a mi solo me a 
![IMAGEN](https://user-images.githubusercontent.com/72475894/194997201-df7dc7b9-1de8-4ac4-b359-5a34a21c75b9.png)
parece uno

**Comment by gumodi35** [comment]
Si tienes que tener los dos, repite las instrucciones, de hecho cuando ya tienes acceso te aparece Misión Backend.

**Comment by jorgelcp05** [comment]
¡Qué tal colega!
Si apareces en el equipo de FrontEnd y realizaste la solicitud de la invitación para el de BackEnd, y aun así no te aparece, en las instrucciones dice que tienes que esperar 1 día, ya que se llegó al límite de solicitudes.
Yo volví a mandar la solicitud de invitación y me apareció que ya estoy en los dos equipos, sin embargo, aún no se refleja, pero hay que esperar:)

**Comment by madridven** [comment]
> Si tienes que tener los dos, repite las instrucciones, de hecho cuando ya tienes acceso te aparece Misión Backend.

Yo he realizado todo y sigo sin la misión BakcEnd

**Comment by miller-vrg** [comment]
Tampoco me aparecen
